application/API/booklist.py

```diff
@@ -23,8 +23,19 @@ class BookLists(Resource):
     def get(self):
         '''Get the info of a book list.'''
         args = parser.parse_args()
-        user_id = args['user_id']
-        list_name = args['list_name']
+        try:
+            user_id = args['user_id']
+        except:
+            return {
+                'message': 'Please provide your user id!'
+            }, 400
+
+        try:
+            list_name = args['list_name']
+        except:
+            return {
+                'message': 'Please provide the book list name!'
+            }, 400
 
         books = db.session.query(Book) \
             .join(BookListToBook, Book.id == BookListToBook.book_id) \
```

Comment:
Maybe also check if there exist a BookList with the user_id and booklist_name?
Do something like:
if not books: return error code and "no mathcing booklist"